editor/src/layout/widgets.rs

```diff
@@ -281,3 +287,17 @@ pub struct RadioEntryData {
 	#[derivative(Debug = "ignore", PartialEq = "ignore")]
 	pub on_update: WidgetCallback<()>,
 }
+
+#[derive(Clone, Serialize, Deserialize, Derivative, Debug, PartialEq)]
+pub struct IconLabel {
+	pub icon: String,
+	#[serde(rename = "gapAfter")]
```

Comment:
Can this not be done in js? It is fine if not, the js constructor things confuse me ;)

Reply:
Which part of it? The rename. It could be but I like having it centralized in one location.

Reply:
Ok